packages/kad-dht/test/kad-dht.spec.ts

```diff
@@ -799,6 +799,37 @@ describe('KadDHT', () => {
 
       expect(res).to.not.be.empty()
     })
+
+    it('should not include itself in getClosestPeers PEER_RESPONSE', async function () {
+      this.timeout(240 * 1000)
```

Comment:
Just curious, why are we using a 240 second timeout?

Reply:
No specific reason, it was copied from the preceding test.